utilities/backupable/backupable_db.cc

```diff
@@ -994,23 +993,26 @@ Status BackupEngineImpl::Initialize() {
     directories.emplace_back(meta_path, &meta_directory_);
     // create all the dirs we need
     for (const auto& d : directories) {
-      auto s = backup_env_->CreateDirIfMissing(d.first);
-      if (s.ok()) {
-        s = backup_env_->NewDirectory(d.first, d.second);
+      IOStatus io_s = backup_env_->GetFileSystem()->CreateDirIfMissing(
```

Comment:
Nit: Grab the FileSystem* out into a temporary variable outside of the loop and use it instead.

Reply:
Should there be a class variable FileSystem* backup_fs_ ?